R/defineModel.R

```diff
@@ -0,0 +1,65 @@
+#' Defining the Model
+#'
+#' Give the complete formula and generate the expected kernel library.
+#'
+#' It processes data based on formula and label_names and creates a kernel
+#' library according to the parameters given in Kern_par.
+#'
+#' @param formula A symbolic description of the model to be fitted.
+#' @param label_names A character string indicating all the interior variables
+#' included in each predictor.
+#' @param data A dataframe to be fitted.
+#' @param Kern_par A dataframe indicating the parameters of base kernels to be
```

Comment:
Add more detailed description about this data frame. For a library of K kernels, how many rows / columns should this data frame have? What does each row / column represent? What are their expected data type? If not enough space, consider add a @description section and describe in detail. 

(This is important because users will use defineModel all the time, and they need to learn to how to specify kernel parameters by reading this function's docstirng)